integrations/gitlab/gitlab_integration/ocean.py

```diff
@@ -10,31 +11,31 @@
 from gitlab_integration.bootstrap import event_handler, system_event_handler
 from gitlab_integration.bootstrap import setup_application
 from gitlab_integration.git_integration import GitlabResourceConfig
-from gitlab_integration.utils import ObjectKind, get_cached_all_services
+from gitlab_integration.utils import (
+    ObjectKind,
+    get_cached_all_services,
+)
 from port_ocean.context.ocean import ocean
 from port_ocean.core.ocean_types import ASYNC_GENERATOR_RESYNC_TYPE
 
 NO_WEBHOOK_WARNING = "Without setting up the webhook, the integration will not export live changes from the gitlab"
 
 
 @ocean.router.post("/hook/{group_id}")
-async def handle_webhook(group_id: str, request: Request) -> dict[str, Any]:
-    event_id = f'{request.headers.get("X-Gitlab-Event")}:{group_id}'
-    with logger.contextualize(event_id=event_id):
-        body = await request.json()
-        await event_handler.notify(event_id, body)
-        return {"ok": True}
+async def handle_webhook(
+    group_id: str, request: Request, background_tasks: BackgroundTasks
+) -> Response:
+    background_tasks.add_task(event_handler.notify, request, group_id)
```

Comment:
Does it work?
Fastapi request sometime isnt ok when u run it from the background 

Can u check that all contexts are still available?